install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -50,6 +53,26 @@ spec:
             - |
               set -e
 
+              echo "Gitpod: Killing any in-progress installations"
+
+              kubectl delete jobs.batch -n {{repl Namespace }} -l component=gitpod-installer,cursor!={{repl Cursor }} --force || true
+              kubectl delete pod -n {{repl Namespace }} -l component=gitpod-installer,cursor!={{repl Cursor }} --force || true
```

Comment:
Maybe a ` --grace-period=0` could be added too, just to be sure?

Reply:
Good call